scripts/docker/buildpack/Dockerfile.emscripten

```diff
@@ -29,6 +29,7 @@
 #   make version=1.39.15 build
 #
 FROM emscripten/emsdk:1.39.15 AS base
+LABEL version="1"
```

Comment:
We should start this with zero in this PR! Then we (1) can check that the build actions properly fail and (2) have the first PR actually building docker files just change this from 0 to 1.